modules/acquia_cms_search/tests/src/FunctionalJavascript/AcquiaFacetSearchTest.php

```diff
@@ -145,6 +145,7 @@ protected function createSampleContent(string $node_type) {
       ]);
     }
     $this->drupalPlaceBlock('facet_block:search_category', ['region' => 'content']);
+    \Drupal::getContainer()->get('search_api.post_request_indexing')->destruct();
```

Comment:
Can we use `$this->container->get('search_api.post_request_indexing')->destruct();`

Reply:
getContainer method is static and with $this->container we won't get a destruct method CI repeats failing.
For example check below screenshot.
<img width="1136" alt="Screenshot 2023-10-04 at 2 38 15 PM" src="https://github.com/acquia/acquia_cms/assets/106175376/a2eb250c-6dff-4534-99f1-5251c9256626">